libfastpm/solver.c

```diff
@@ -618,7 +639,7 @@ fastpm_set_species_snapshot(FastPMSolver * fastpm,
     /* convert units */
 
     /* potfactor converts fastpm Phi to dimensionless */
-    double potfactor = 1.5 * c->OmegaM / (HubbleDistance * HubbleDistance);
+    double potfactor = 1.5 * c->Omega_cdm / (HubbleDistance * HubbleDistance);  // FIXME: Change to Omege_m(a) in neutrino run?
```

Comment:
Is this settled?

Reply:
I'm not 100% sure what this does because I don't think I ever compute the potential/tidal. But I guess now that all neutrinos are matter-like we want this to go back to c->Omega_m?